.github/workflows/certification.yml

```diff
@@ -24,8 +24,6 @@ on:
       - 'release-*'
   pull_request:
     branches:
-      # TODO: REMOVE "master" BEFORE MERGING
-      - 'master'
```

Comment:
Let's only remove the comment please. We don't know why the comment was there.

There is no harm in having master here -- and it's just too painful to merge release back into master everytime we have a release / hotfix etc otherwise